site/docs/v1/tech/apis/scim/SCIMServiceProvider.adoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: doc
+title: SCIM Service Provider API Overview
+description: APIs for dsicovering configuration information about FusionAuth SCIM Service Provider using your SCIM Client
+---
+
+This API is used to retrieve information about the configuration of FusionAuth SCIM Service Provider per the SCIM specification.
+
+https://datatracker.ietf.org/doc/html/rfc7644
+
+[method]#GET# [uri]#/api/scim/resources/v2/Schemas#
+--
+[method]#GET# [uri]#/api/scim/resources/v2/ServiceProviderConfig#
+--
+[method]#GET# [uri]#/api/scim/resources/v2/ResourceTypes#
+
+Documentation in progress...
```

Comment:
```suggestion
```

Just leave it blank. You can also ship it but don't include a link anywhere if you must.